File: Modulo04/AlunoProfessor-Java/Aluno.java
public class Aluno extends Pessoa {
    private int RA;

    public Aluno(String nome, int idade, int RA) {
        super(nome, idade);
        this.RA = RA;
    }

    public int getRA() {
        return this.RA;
    }

    public void setRA(int RA) {
        this.RA = RA;
    }
    
    public void imprime() {
        super.imprime();
        System.out.println("RA: " + this.RA);
    }
}

File: Modulo04/AlunoProfessor-CSharp/Aluno.cs
using System;
public class Aluno : Pessoa {
    private int RA;

    public Aluno(String nome, int idade, int RA) : base (nome, idade) {
        this.RA = RA;
    }

    public int getRA() {
        return this.RA;
    }

    public void setRA(int RA) {
        this.RA = RA;
    }
    
    override public void imprime() {
        base.imprime();
        Console.WriteLine("RA: " + this.RA);
    }
}